README.md

```diff
@@ -36,22 +92,22 @@ services:
       - 5678:5678
 ```
 
-With `docker app` [installed](#installation) let's create an Application Package
-based on this Compose file:
+Next, create an App definition using the `docker app init` command:
 
 ```console
-$ docker app init hello
+$ docker app init --compose-file docker-compose.yml hello
+Created "hello.dockerapp"
 $ ls
```

Comment:
You can use the `tree` command here
```
$ tree
.
├── docker-compose.yml
└── voting-app.dockerapp
    ├── docker-compose.yml
    ├── metadata.yml
    └── parameters.yml

1 directory, 4 files
```
And maybe also add a note that the first `docker-compose.yml` file is not used anymore?

Reply:
done